package.json

```diff
@@ -15,7 +15,7 @@
     "es6-promise": "^3.0.2",
     "es6-shim": "^0.35.1",
     "expect.js": "~0.2.0",
-    "glob": "~3.2.7",
+    "glob": "~4.2.0",
```

Comment:
Any reason for this upgrade?  Any major dependency upgrade requires a lot of testing since it might have breaking changes :)